pointgrey_camera_driver/cfg/PointGrey.cfg

```diff
@@ -109,8 +111,11 @@ gen.add("tilt",               int_t, 	SensorLevels.RECONFIGURE_RUNNING,     "Con
 
 gen.add("brightness",         double_t, SensorLevels.RECONFIGURE_RUNNING,     "Black level offset.",				                                             0.0, 0.0, 10.0)
 
+gen.add("sharpness",         double_t, SensorLevels.RECONFIGURE_RUNNING,     "sharpness.",				                                             1024.0, 0.0, 4095.0)
+gen.add("saturation",         double_t, SensorLevels.RECONFIGURE_RUNNING,     "saturation.",				                                              100.0, 0.0, 100.0)
```

Comment:
Please ensure that these default values (and the ones above) correspond to what the driver used to do by default (so that no one will be surprised when upgrading).